content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,75 @@
+
+## Traffic Management
+
+- **Added** the capability for Istio to automatically determine protocol types avoiding the need for specifying the mandatory container ports field in the pod spec.
+- **Added** a mode to the Gateway API for mutual TLS operation that is capable of reading certificates via SDS.
+- **Added** enhancements to the `EnvoyFiltier` API such that all configuration can be added or updated.
```

Comment:
nit: `EnvoyFilter` instead of `EnvoyFiltier`

Reply:
addressed.